### ISSUE TITLE
[Ch6: Roles] Role creation

### ISSUE THREAD

**Issue by thescouser89** [geerlingguy/ansible-for-devops#227]
Hi,

On page 129 on the pdf, it says that to scaffold a role, this command is run:
```
ansible-galaxy init role_name
```

On F31, I need to invoke it as such:
```
ansible-galaxy role init role_name
```

**Comment by geerlingguy** [comment]
This still works the same in Ansible 2.9.x and below, but you're correct in that Ansible devel / 2.10 and later will require the `role` subcommand to work with roles explicitly.

I'm planning on reworking parts of Chapter 6 more extensively later when I deal with collections (see #202), but for now I've gone ahead and set all the previously un-subcommand-ed commands to have `role` (e.g. `ansible-galaxy install` is now `ansible-galaxy role install` when dealing with only roles).

Thanks for filing the issue and I'll credit you with this fix in version 1.23 of the book!

**Comment by thescouser89** [comment]
Thank you! :)
